server/api.js

```diff
@@ -30,9 +31,118 @@ router.get("/energisers", (req, res) => {
     }).catch((err) => {
         console.log(err);
         res.status(500).send(err);
+=======
+// router.get("/", (_, res) => {
+// 	res.json({ message: "Hello, world!" });
+// });
+
+
+///=================POST REQUESTS==================///
+
+//  ADD AN ENERGISER
+router.post("/energisers", function (req, res) {
+  let energiserTitle = req.body.title;
+  let energiserImage = req.body.image;
+  let energiserDescription = req.body.description;
+  let energiserLink = req.body.link;
+
+//    || !energiserImage || !energiserLink
+
+  if (!energiserTitle  || !energiserDescription) {
+    return res
```

Comment:
Good to see you thinking about validation

Reply:
Thank you